packages/framework/src/Controller/Admin/BillingAddressController.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Controller\Admin;
+
+use Shopsys\FrameworkBundle\Component\Domain\AdminDomainTabsFacade;
+use Shopsys\FrameworkBundle\Form\Admin\Customer\BillingAddressAndRelatedInfoFormType;
+use Shopsys\FrameworkBundle\Model\AdminNavigation\BreadcrumbOverrider;
+use Shopsys\FrameworkBundle\Model\Customer\BillingAddressDataFactory;
+use Shopsys\FrameworkBundle\Model\Customer\BillingAddressFacade;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class BillingAddressController extends AdminBaseController
+{
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\AdminNavigation\BreadcrumbOverrider $breadcrumbOverrider
+     * @param \Shopsys\FrameworkBundle\Component\Domain\AdminDomainTabsFacade $adminDomainTabsFacade
+     * @param \Shopsys\FrameworkBundle\Model\Customer\BillingAddressDataFactory $billingAddressDataFactory
+     * @param \Shopsys\FrameworkBundle\Model\Customer\BillingAddressFacade $billingAddressFacade
+     */
+    public function __construct(
+        protected readonly BreadcrumbOverrider $breadcrumbOverrider,
+        protected readonly AdminDomainTabsFacade $adminDomainTabsFacade,
+        protected readonly BillingAddressDataFactory $billingAddressDataFactory,
+        protected readonly BillingAddressFacade $billingAddressFacade,
+    ) {
+    }
+
+    /**
+     * @param \Symfony\Component\HttpFoundation\Request $request
+     * @param int $id
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    #[Route(path: '/billing-address/edit/{id}', requirements: ['id' => '\d+'], name: 'admin_billing_address_edit')]
```

Comment:
Fixed in

- https://github.com/shopsys/shopsys/pull/3261/commits/99dafefcc428cdcbbdea2def65e80bc35739ef91
- https://github.com/shopsys/shopsys/pull/3261/commits/9f5742b17e4be7e460eca716f8004b4864f5deb6
- https://github.com/shopsys/shopsys/pull/3261/commits/c4c9e7e291c575d57c6e020ce76c200c2ab25e34
- https://github.com/shopsys/shopsys/pull/3261/commits/e15dc508b44981c873c51b8cb4aa20caa1293ff4
- https://github.com/shopsys/shopsys/pull/3261/commits/64da44dcc8eb3707eff6c583a1a2caabf4fdf767
- https://github.com/shopsys/shopsys/pull/3261/commits/41de0db7cad97343197fef8152bcec5fa2557e68
- https://github.com/shopsys/shopsys/pull/3261/commits/82438201fa472193bdba9ad876d6f3775e798f6e

I checked others controllers and fixed order as well
- https://github.com/shopsys/shopsys/pull/3261/commits/c996a867a124c65ef7c3648d1262dc770fc4c07d

I spent some time with looking for existing fixer, that I would like to have. I don't want to think about it during implementing and CR. Unfortunately I was not able to find any, so I it gave it up. :disappointed:

Reply:
Thanks!
I had the same idea about fixer, but unfortunately, it seems like it may be an overly complex task for a mere token-based standard to check... 
Maybe the custom PHPStan rule can warn about this... I have added this point to internal discussion